policies/PrivacyPolicy.html

```diff
@@ -469,8 +470,61 @@ <h3><strong>Information automatically collected</strong></h3>
                                           </ul>
                                           <div><h2></h2></div>
                                           <div><br /></div>
+                                          <div id="thirdpartysources">
+                                            <strong><h2>5. WHEN AND HOW DO WE RECEIVE INFORMATION FROM THIRD PARTIES?</h2></strong>
+                                          </div>
+                                          <div><br /></div>
+                                          <div>
+                                            <strong><em>In Short:</em></strong><em> We may receive information from third parties from time to time, for example, in order to fulfill moderation tickets.</em> 
```

Comment:
```suggestion
                                            <strong><em>In Short:</em></strong><em> We may receive information from third parties, for example, in order to fulfil moderation tickets.</em> 
```

Please do a spellcheck pass